brigade.js

```diff
@@ -27,11 +27,11 @@ events.on("exec", (e, p) => {
   ];
 
   // This job renders helm.sh
-  const buildHelmSh = new Job("helm-sh", "node:9");
+  const buildHelmSh = new Job("helm-sh", "ruby:2.3");
   buildHelmSh.timeout = timeout
   buildHelmSh.storage.enabled = true
   buildHelmSh.tasks = [
-    "apt-get update -y && apt-get install -yq ruby2.2 ruby2.2dev",
+    "apt-get update -y && apt-get install -yq ruby ruby-dev",
```

Comment:
Now that you are starting with ruby, you need to change this line to install node

I think this should work:
```
curl -sL https://deb.nodesource.com/setup_9.x | bash -
apt-get install -y nodejs
```

Reply:
Thanks @adamreese, done!